webhook-connector/order-events.md

```diff
@@ -0,0 +1,131 @@
+---
+description: Order Events overview.
+---
+
+# Order Events
+
+When using the order webhook you may configure and receive below order events.
+
+## Order is issued or updated by buyer
+
+The buyer can issue new order lines or update existing issued lines.
+
+If confirmed delivery schedule or prices are updated, this will become a reopen request.
+
+| OrderEvent                  | Webhook Configuration |
+| --------------------------- |-----------------------|
+| `OrderIssuedByBuyer`        | New order is issued by the buyer |
+| `OrderLinesIssuedByBuyer`   | Additional order lines are added by the buyer |
+| `OrderLinesReissuedByBuyer` | Updated order lines are reissued by the buyer |
+| `OrderLinesUpdatedByBuyer`  | Order and line fields other than prices & delivery schedules are updated by the buyer |
+
+## Order confirmed by supplier or buyer
+
+The supplier can either accept, reject or propose an alternative (see next section).
+
+* Accepted means that the supplier confirmed the order line(s) as requested by the buyer.
+* Rejecting means that the supplier cannot deliver the order line(s) at all.
+* Confirmed by supplier happens when the supplier confirms and [Auto Confirm](https://docs.tradecloud1.com/api/processes/order/buyer/issue/indicators#auto-confirm) has been enabled by the buyer.
+* Confirmed by buyer is used when the buyer migrates already confirmed order lines to Tradecloud.
+
+| OrderEvent                      | Webhook Configuration                     |
+| ------------------------------- | ----------------------------------------- |
+| `OrderLinesAcceptedBySupplier`  | Order lines are accepted by the supplier  |
+| `OrderLinesRejectedBySupplier`  | Order lines are rejected by the supplier  |
+| `OrderLinesConfirmedBySupplier` | Order lines are confirmed by the supplier |
+| `OrderLinesConfirmedByBuyer`    | Order lines are confirmed by the buyer    |
+
+
+## Order proposal by supplier
+
+The supplier can propose an alternative delivery schedule or prices.
+
+The buyer either approves or rejects the proposal.
+
+| OrderEvent                            | Webhook Configuration                                        |
+| --------------------------------------| ------------------------------------------------------------ |
+| `OrderChangesProposedBySupplier`      | Order changes are proposed by the supplier                   |
+| `OrderChangesProposalApprovedByBuyer` | By supplier proposed order changes are approved by the buyer |
+| `OrderChangesProposalRejectedByBuyer` | By supplier proposed order changes are rejected by the buyer |
+
+## Order updated by supplier
+
+The supplier can update existing order lines.
+
+If delivery schedule or prices are updated this will become either a proposal or reopen request.
```

Comment:
```suggestion
If delivery schedule or prices are updated this will become either a [proposal](#order-proposal-by-supplier) or [reopen request](#order-reopen-request-by-supplier).
```

Reply:
Updated